reckoner/reckoner.py

```diff
@@ -85,7 +86,23 @@ def install(self, charts=[]):
 
         """
         selected_charts = charts or [chart._release_name for chart in self.course.charts]
-        return self.course.plot(selected_charts)
+        try:
+            self.course.plot(selected_charts)
+        except NoChartsToInstall as error:
```

Comment:
probably could use a test here to assure this function can handle this exception